index.html

```diff
@@ -48,8 +49,20 @@ <h2></h2>
       </section>
     </main>
 
-    <footer>
-    </footer>
 
+<!--FOOTER, BUILT BY SHALINI-->
+    <footer >
+      <div class="container-fluid websiteFooter">
+        <div class="col-sm-6">
+        <img src="resources/images/logo.png">
```

Comment:
`<img>` elements need alt attributes.

Reply:
Thanks a lot @camper-fcc for the review. I was sure I wouldn't have got it right for the first time. I am working on all your remarks for my next pull request. However for now Ill not do any changes for Bootstrap, Ill wait until for our discussion on the usage of Bootstrap.

Reply:
@shalinidc Ok, cool. Are you planning to remove the `.DS_Store` files from and add the alt attribute on the image element for this pull request, or should we close this one and you'll make a new one? Those are the main things I'd like to see before approving this pull request. It looks like we may not be using Bootstrap (depending on Willa's response), so keep that in mind too.

Reply:
@camper-fcc Yes I wasn't aware about the .DS_Store, I will take care of it in my pull request including the alt attribute. The only thing I wasn't sure was about the pushing the changes from the forked repo's staging branching, can you please let me know whats the best practice to do ?
I'll keep in mind about usage of Bootstrap, I agree to to your reasons for not using the bootstrap.
We can go ahead and close this one and I would submit a new pull request.

Reply:
For our project, we agreed to work in a feature branch in our forked repo (https://github.com/freeCodeCamp/freeCodeCamp/blob/staging/CONTRIBUTING.md#important-always-edit-on-a-branch):

> Important: *ALWAYS EDIT ON A BRANCH*
> Take away only one thing from this document: Never, EVER make edits to the staging branch. ALWAYS make a new branch BEFORE you edit files. This is critical, because if your PR is not accepted, your copy of staging will be forever sullied and the only way to fix it is to delete your fork and re-fork.

I'll close this pull request so you can make the pull request from another branch.